mappings/net/minecraft/unmapped/C_kkthincp.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/unmapped/C_kkthincp
```

Comment:
Is there a reason this is un-named?

Reply:
I couldn't think of a name for it, since the mojmap name is `DimensionTypes`, and we already have a `DimensionTypes` class